.github/workflows/scripts/prepare-upgrade-test.sh

```diff
@@ -47,9 +50,9 @@ deploy_central() {
 
     kubectl rollout status deployment central -n stackrox --watch --timeout 5m
 
-    ROX_ADMIN_PASSWORD="$(cat bundle-test1/password)"
-    echo "::add-mask::$ROX_ADMIN_PASSWORD"
-    export ROX_ADMIN_PASSWORD
+    ROX_PASSWORD="$(cat bundle-test1/password)"
+    echo "::add-mask::$ROX_PASSWORD"
+    export ROX_PASSWORD
```

Comment:
Sure about this change? I see references to `ROX_ADMIN_PASSWORD` in the following functions.

Reply:
Do I understand correctly, that this PR is not working properly without https://github.com/stackrox/test-gh-actions/pull/192? Then maybe let's merge them together into this repo?

Reply:
Good catch, this change shouldn't have gone in. Fixed in https://github.com/stackrox/stackrox/pull/14765/commits/635eac12c4d9a5fb5d01b99b75b299c3cf0527d2

Reply:
https://github.com/stackrox/test-gh-actions/pull/192 brings changes from 
* https://github.com/stackrox/stackrox/commit/b5beed513133fba7803f71f94a2834d5b3959e7d
* https://github.com/stackrox/stackrox/commit/1cbbf6990c5762d42bb04f99d3ff231ac2e22484

back to the `test-gh-actions` repository, that were not added there first.